app/models/anime.rb

```diff
@@ -137,7 +135,10 @@ def self.fuzzy_find(title)
   end
 
   after_save do
-    episodes.create_defaults(episode_count) if
-      episode_count_changed? && episodes.length != episode_count
+    if episode_count_guess_changed? && episodes.length != episode_count_guess
+      episodes.create_defaults(episode_count_guess || 0)
+    elsif episode_count_changed? && episodes.length != episode_count
+      episodes.create_defaults(episode_count || 0)
+    end
```

Comment:
Won't this order cause the guess to be used instead of the count if they're both set (which ideally they shouldn't be, but it's possible they will be)

Reply:
they won't ever be both set because we nil out the guess in `before_save` if count is set.